includes/wikia/tests/GlobalTitleTest.php

```diff
@@ -165,4 +190,14 @@ public function stripArticlePathDataProvider() {
 			['/wiki/Ludovic_Hindman/subpage','/wiki/$1','Ludovic_Hindman/subpage'],
 		];
 	}
+
+	public function mainPageDataProvider() {
+		return [
+			['Whatever', true, 'http://community.wikia.com/wiki/Whatever'],
+			['Namespace:Test', true, 'http://community.wikia.com/wiki/Namespace:Test'],
+			['Namespace with spaces:Test', true, 'http://community.wikia.com/wiki/Namespace_with_spaces:Test'],
+			['Namespaces_with_underlines:Test', true, 'http://community.wikia.com/wiki/Namespaces_with_underlines:Test'],
+			['Notexists', false, 'http://community.wikia.com/wiki/Main_Page']
```

Comment:
why there is a "notexists" here? how it differs from the previous cases?

Reply:
it has `false` as a second attribute which mocks return value for `exists` method.